### PR TITLE
docker-system, docker-compose: add Japanese translation, update page

### PR DESCRIPTION
**Pull request by Kumamoto-Hamachi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ja/common/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- 全てのコンテナを起動し、必要に応じて再ビルドする:
+
+`docker compose up --build`
+
+- 代替composeファイルを使って全てのコンテナを起動する:
+
+`docker compose --file {{ファイルパス}} up`
+
+- 全てのコンテナを停止する:
```
Comment (kbdharun) [inline pages.ja/common/docker-compose.yml:22]:
```suggestion
- 実行中のすべてのコンテナーを停止します:
```
Shouldn't it be something like this? (Stop all running containers)?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. Just a tiny suggestion and the PR is GTG.

File: pages.ja/common/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+> 複数コンテナを持つDockerアプリケーションの実行と管理をします。
+> 詳しくはこちら: <https://docs.docker.com/compose/reference/>.
+
+- 実行中のコンテナ全てをリスト表示する:
+
+`docker compose ps`
+
+- カレントディレクトリにある `docker-compose.yml`ファイルを使用してバックグラウンドで全てのコンテナを作成・起動する:
+
+`docker compose up -d`
```
Comment (EmilySeville7cfg) [inline pages.ja/common/docker-compose.yml:12]: Long option found:

```suggestion
`docker compose up --detach`
```

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages.ja/common/docker-compose.yml (hunk 2 above)
Comment (kbdharun) [reply pages.ja/common/docker-compose.yml:12]: @Kumamoto-Hamachi If the above suggestion is applied, same must be updated in English pages too to be uniform.

Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker-compose.yml:12]: Thank you both for your reviews.
I will change the PR title to "docker-system, docker-compose: add Japanese translation, update page" and also change the docker-compose.md file for the English version as reviewed.

File: pages.ja/common/docker-compose.yml (hunk 1 above)
Comment (Kumamoto-Hamachi) [reply pages.ja/common/docker-compose.yml]: Surely！ :bow: 
Fixed in https://github.com/tldr-pages/tldr/pull/9828/commits/c0a9e330427d9d5aeaf49b322b1f4376de391a9e

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM updated the wrong file extension and changed to the long flag on the other pages.

**Pull request merged**